src/backend/howlongtobeat/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { HowLongToBeatEntry, HowLongToBeatService } from 'howlongtobeat'
+
+export async function getHowLongToBeat(
+  title: string
+): Promise<HowLongToBeatEntry> {
+  return new Promise((resolve) => {
+    const hltb = new HowLongToBeatService()
+    hltb.search(title).then((results) => {
+      resolve(results[0])
+    })
```

Comment:
should we add a `catch` function here? in case the API is down or there's no internet connection

Reply:
Done